draft-regext-rdap-extensions.md

```diff
@@ -181,16 +239,74 @@ The following example shows this use with a JSON object.
       ],
       "lunarNIC_author" :
       {
-        "firstInitial": "J",
+        "firstInitial": "R",
         "lastName": "Heinlein"
       }
     }
 
-Here the JSON name "lunarNic_author" will separate the JSON from other
+Here the JSON name "lunarNIC_author" will separate the JSON from other
 extensions that may have an "author" structure. But the JSON contained
 within "lunarNIC_author" need not be prepended as the extension collision
 is avoided by "lunarNIC_author".
 
+## Object Classes in Extensions {#object_classes_in_extensions}
+
+As described in [@!RFC9082] and #(usage_in_queries), an extension may define new paths in URIs.
+If the extension describes the behavior of an RDAP query using that path to return a new RDAP
+object classs, the JSON names are not required to be prepended with the extension identifier
+as described in (#child_json_values). However, the extension MUST define the value for the
+`objectClassName` string (which is used by clients to evaluate the type of the response), 
+and that value MUST be prepended with the extension identifier in
+the same manner as would be required for JSON names (to avoid collisions with object classes 
+defined in other extensions).
```

Comment:
What about having objectClassName be the bare extension identifier?  That doesn't appear to pose any collision problems or similar.  (I don't have a strong opinion on this, but it seems like a reasonable thing to do that some people might find useful.)